configure/RELEASE

```diff
@@ -24,22 +24,26 @@ TEMPLATE_TOP=$(EPICS_BASE)/templates/makeBaseApp/top
 # If using the sequencer, point SNCSEQ at its top directory:
 #SNCSEQ=$(EPICS_BASE)/../modules/soft/seq
 
-SUPPORT=/corvette/home/epics/devel
+#SUPPORT=/corvette/home/epics/devel
```

Comment:
These changes are specific to your site and should not be merged.  Please checkout master, make local edits, but do not commit and have as part of your PR.

Reply:
OK